kpops/pipeline_generator/pipeline.py

```diff
@@ -36,11 +37,21 @@ class ValidationError(Exception):
     pass
 
 
+class InternalNodeRepresentation(BaseModel):
+    name: str
+    component: Optional[PipelineComponent]
+    is_topic: bool
```

Comment:
When we treat topics as components this will go away!

Reply:
What do you mean ?

Reply:
https://github.com/bakdata/kpops/issues/351#issuecomment-1708003817

Reply:
so, we refactor when we are there, right ?

Reply:
Yeah don't worry about it now :)